frontend/src/pages/SubProjects/SubProjectTable.js

```diff
@@ -102,6 +113,7 @@ const displaySubprojectBudget = budgets => {
 };
 
 const getTableEntries = (
+  idsPermissionsUnassigned,
```

Comment:
since we have a lot of parameters now, we should think about wrapping them in an object instead